language/tools/move-mv-llvm-compiler/src/stackless/llvm.rs

```diff
@@ -761,14 +761,17 @@ impl Builder {
         }
     }
 
-    // TODO: If \p name isn't provided get a tempname.
     pub fn build_binop(
         &self,
         op: LLVMOpcode,
         lhs: AnyValue,
         rhs: AnyValue,
         name: &str,
     ) -> AnyValue {
+        assert!(
+            name.trim().len() == name.len(),
+            "Invalid name provided for identifier"
+        );
```

Comment:
Nit: I would remove this assert-- an empty name is not an error.
Otherwise, LGTM.